src/DynamoCoreWpf/UI/Prompts/FunctionNamePrompt.xaml.cs

```diff
@@ -19,9 +18,6 @@ public FunctionNamePrompt(IEnumerable<string> categories)
         {
             InitializeComponent();
 
-            Owner = WpfUtilities.FindUpVisualTree<DynamoView>(this);
```

Comment:
Since this is not release critical, I would like to ask, was the bug exist because the Owner was not set properly or set to Revit Window by accident?

Reply:
No, I actually could not figure out why this is happening. I applied a post-fix. It is visible that the modal is set to the center of Dynamo at the start, then jumps to a strange location. I could not figure out what was causing this behavior. I compared with the Preferences view for all relevant lines where we set the location and could not make any difference. I spent maybe 3 hours trying to figure out this particular issue. 
If we have time, I can have a look tomorrow with fresh eyes, I just lost the battle today.

Reply:
yeah, no worries, fine to come back on this later

Reply:
I was able to found the real culprit, @QilongTang ! We were assigning the `Owner` twice, actually, the initial one was assigned during Dynamo initialization, I suppose. That was messing up with the window positioning. Removing this resolved the issue!
I cleaned up the previous fix and pushed the new code.